docs/components/README.md

```diff
@@ -1,3 +1,3 @@
 # Component Docs
 
-Documents in this directory are engineering guidance for building components that are not included in the `material.dart` library in Flutter.
\ No newline at end of file
+Documents in this directory are engineering guidance for components in the `material.dart` library in Flutter (best viewed on [material.io](https://material.io/develop/flutter/components/app-bars-bottom), as well as components not included in the library (e.g. backdrop).
```

Comment:
Is bottom app bar the best one to use? Maybe FAB or buttons or top app bar?

Reply:
Sure, FABs it is